stablehlo/dialect/StablehloOps.h

```diff
@@ -77,6 +77,10 @@ class TokenType : public Type::TypeBase<TokenType, Type, TypeStorage> {
   using Base::Base;
 };
 
+// Verifies the source target pairs attached to collective permute.
+LogicalResult verifyBroadcastSourceTargetPairs(Operation *op,
```

Comment:
I think this method no longer exists? Ctrl+f in review shows nothing at least

Reply:
Oh, yes that is my bad. No idea why I made this...